draft-yoshino-wish.txt

```diff
@@ -238,6 +238,11 @@ Internet-DrafWiSH: A General Purpose Message Framing over B October 2016
    for flushing part of a large message payload without waiting for the
    total size of the message to be determined.
 
+   The CMP bit is used when a compression is in use.  Otherwise, it MUST
```

Comment:
... is used when compression is enabled for the current message. Otherwise, the CMP bit MUST ..